docs/get-started/onnx/index.md

```diff
@@ -0,0 +1,249 @@
+the same random input. If the outputs differ an assertion error is raised.
+
+Now that we have our model exported and have verified that it has the same behavior as the
+original, it's time to deploy it 🚀.
+
+
+## Deploying using the ONNX Encoder
+
+You have already finetuned your own model and transferred it to ONNX format. Let's start deploying in the Jina flow. If you are not familiar with Jina-Hub or Jina flow, check this:
+[Use Hub Executor](https://docs.jina.ai/advanced/hub/use-hub-executor/)
+[Use Jina Flow](https://docs.jina.ai/fundamentals/flow/)
+
+Here are the steps you can follow:
+
+### Add ONNXEncoder to Jina flow
+
+We already have an ONNX encoder on Jina-Hub, let's add `ONNXEncoder` to Jina flow:
+
+using docker image:
+
+```python
+from jina import Flow
+
+f = Flow().add(uses='jinahub+docker://ONNXEncoder',
+                uses_with={'model_path': 'tuned-model.onnx'})
+```
+
+or using source code:
+
+```python
+from jina import Flow
+
+f = Flow().add(uses='jinahub://ONNXEncoder',
+                uses_with={'model_path': 'tuned-model.onnx'})
+```
+
+`model_path` is the path of the ONNX model you exported.
+
+### Complete the flow by adding indexer and starting the service
+
+```python
+from jina import DocumentArray, Executor, requests, Flow
+from typing import Optional
+import numpy as np
+
+
+class SimpleIndexer(Executor):
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self._dam = DocumentArray()
+
+    @requests(on='/index')
+    def index(self, docs: DocumentArray, **kwargs):
+        self._dam.extend(docs)
+
+    @requests(on='/search')
+    def search(self, docs: DocumentArray, **kwargs):
+        docs.match(self._dam)
+        
+
+f = Flow(port_expose=12345, 
+         protocol='http').add(uses='jinahub://ONNXEncoder', 
+                              uses_with={'model_path': 'tuned-model.onnx'},
+                              name='Encoder').add(uses=SimpleIndexer, 
+                                                  name='Indexer')
+        
+with f:
+    f.post('/index', data, show_progress=True)
+    f.block()
+```
+
+You will see something like this:
+
+```bash
+    Flow@6260[I]:🎉 Flow is ready to use!                                           
+    🔗 Protocol:            HTTP
+    🏠 Local access:        0.0.0.0:12345
+    🔒 Private network:     192.168.31.213:12345
+    💬 Swagger UI:          http://localhost:12345/docs
+    📚 Redoc:               http://localhost:12345/redoc
+⠼       DONE ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╸ 0:00:14 100% ETA: 0 seconds 80 steps done in 14 seconds
+```
+
+which means that the service has been successfully started.
+
+### Start to query!
+
+Finally it's time to see what we can get from our service. Start a client and try to query using the service we have just built. 
+
+```python
+from jina import Client, Document, DocumentArray
+from jina.types.request.data import Response
```

Comment:
from docarray import ...

we don't have `Response` class anymore, have you tried on your machine?

`pip install --pre jina`